spec/workers/enqueue_subject_queue_worker_spec.rb

```diff
@@ -53,19 +72,54 @@
       end
     end
 
-    describe "#load_subjects" do
+    context "when selecting via strategy param" do
+      let(:result_ids) { [1] }
+
+      it 'should fallback from the override strategy' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.